addons/mail/static/src/new/web/messaging_menu/messaging_menu.js

```diff
@@ -45,7 +45,7 @@ export class MessagingMenu extends Component {
      * @returns Thread types matching the given tab.
      */
     tabToThreadType(tab) {
-        return tab === "chats" ? ["chat", "group"] : tab;
+        return tab === "chat" ? ["chat", "group"] : tab;
```

Comment:
I've seen another pending PR where the last value `tab` was transformed to `[tab]`, isn't it needed here? For consistency of return value.